src/main/java/fr/techad/edc/popover/internal/swing/builder/ContextualTitleComponentBuilderImpl.java

```diff
@@ -56,31 +76,73 @@
     }
 
     @Override
-    public JComponent build() {
+    public ContextualTitleComponentBuilder<JComponent> setErrorBehavior(ErrorBehavior errorBehavior) {
+        this.errorBehavior = errorBehavior;
+        return this;
+    }
+
+    @Override
+    public ContextualTitleComponentBuilder<JComponent> setLanguageCode(String languageCode) {
+        this.languageCode = languageCode;
+        return this;
+    }
+
+    @Override
+    public JComponent build() throws InvalidUrlException, IOException {
```

Comment:
In ContextualContentComponentBuilderImpl, you catch exception. Why do you expose them here?
Catch exception too...